02_ml_models/README.md

```diff
@@ -0,0 +1,77 @@
+# Using a Machine Learning Model
+
+## Session 1: Introduction to ml5.js and pre-trained models
+
+### Objectives:
+* Understand the concept of a “machine learning model.”
+* What is a “pre-trained model”?
+* What does it mean to discuss the “architecture” of a machine learning model?
+* Define and diagram an artificial neural network.
+* Understand what ml5.js is and how it fits into the TensorFlow and open source machine learning library ecosystem.
+* Learn how to create an image classifier with ml5.js and MobileNet.
+* Understand how the MobileNet model was trained, specifically the origins and collection methodology for the training.
+
+### ml5.js video tutorials
+* [A Beginner's Guide to Machine Learning with ml5.js - video tutorial](https://youtu.be/jmznx0Q1fP0?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Image Classification with MobileNet - video tutorial](https://youtu.be/yNkAuWz5lnY?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Webcam Image Classification - video tutorial](https://youtu.be/D9BoBSkLvFo?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Transfer Learning with Feature Extractor - video tutorial](https://youtu.be/kRpZ5OqUY6Y?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Feature Extractor Classification - video tutorial](https://youtu.be/eeO-rWYFuG0?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+* [ml5.js: Feature Extractor Regression - video tutorial](https://youtu.be/aKgq0m1YjvQ?list=PLRqwX-V7Uu6YPSwT06y_AEYTqIwbeam3y)
+
+### ml5.js resources
+* [ml5 website](https://ml5js.org)
+* [ml5 web editor examples](https://github.com/ml5js/ml5-examples/)
+
+### ml5 code editor examples
+* [Drag and Drop Image Classification](https://editor.p5js.org/ml5/sketches/S1PLinE27)
```

Comment:
This link is broken, and I can't find this example in the p5 editor ml5 account...

Reply:
If we need to make one, let me know!

Reply:
I made this example last year as a band-aid for when we were having trouble with `loadImage()` and the p5 web editor. I can probably track it down / remake it really quickly, I'll open a separate issue as I plan to merge this soon!

Reply:
See #19